From dcdf5c66aec43f989d7274f3bea9d494c9020694 Mon Sep 17 00:00:00 2001
From: Nehal J Wani <nehaljw.kkd1@gmail.com>
Date: Sat, 23 Jun 2018 11:45:31 +0000
Subject: [PATCH 2/2] Pick python using /usr/bin/env instead of relying on
 sys.executable

sys.executable can point to the python in the host prefix, which can
be greater than BINPRM_BUF_SIZE and cause trouble
---
 .../intel_decimal128_readtest_wrapper.py.in                     | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/third_party/IntelRDFPMathLib20U1/intel_decimal128_readtest_wrapper.py.in b/src/third_party/IntelRDFPMathLib20U1/intel_decimal128_readtest_wrapper.py.in
index 24585cdd15..9d9a596937 100755
--- a/src/third_party/IntelRDFPMathLib20U1/intel_decimal128_readtest_wrapper.py.in
+++ b/src/third_party/IntelRDFPMathLib20U1/intel_decimal128_readtest_wrapper.py.in
@@ -1,4 +1,4 @@
-#!@readtest_python_interpreter@
+#!/usr/bin/env python
 """
 Wrapper python script to launch the readtest_program program as interpolated
 by scons with the standard input set to the test file readtest_input, also as
-- 
2.17.0

